content_resolver/config_manager.py

```
@@ -929,8 +970,10 @@ def get_configs(self):
             try:
                 try:
                     json_data = self.load_data(os.path.join(directory, json_file))
-                except Exception as err:
-                    raise ConfigError(f"Error loading a JSON data file '{json_file}': {err}") from err
+                except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)